addins/Cake.AssemblyInfoReflector.yml

```diff
@@ -5,5 +5,6 @@ Assemblies:
 Repository: https://github.com/wexman/Cake.AssemblyInfoReflector
 Author: wexman
 Description: "Cake addin for retrieving assembly info from an existing assembly."
+Prerelease: "true"
```

Comment:
I had this debate with @gep13. I didn't think we should include pre-release addins but he was adamant.

Reply:
Including pre-release is one thing but should we use pre-release if it's been released?

Reply:
@devlead I think my thought here was that if this automatic auditing works the way that we want it to, when a released version is available, it will quite quickly be updated to reflect that on the site.  So switching it to be marked as pre-release made sense in my head.  Otherwise, what would determine when this gets switched?  If left to a manual effort, it will likely never get changed.

Reply:
The audit tools runs every 12 hours so it's able to notice quickly when a new version of a given addin is released and to automatically generate the updated YAML and submit a PR with changes. However, merging the resulting PR is a manual process which can take quite some time if the cake team happens to be quite busy.

Reply:
Once we have this round of PR’s finished, and we are all happy with the process, I think getting PR’s merged and pushed out should be something reasonably quick to get done.